criu/files-reg.c

```diff
@@ -1792,30 +1792,45 @@ static int linkat_hard(int odir, char *opath, int ndir, char *npath, uid_t uid,
 	return ret;
 }
 
-static void rm_parent_dirs(int mntns_root, char *path, int count)
+static int rm_parent_dirs(int mntns_root, char *path, int count)
 {
 	char *p, *prev = NULL;
+	int ret = -1;
 
 	if (!count)
```

Comment:
This if statement is not useful (the while loop condition is sufficient), but I don't think it matters.

Reply:
Thanks! Let's remove it.